### ISSUE TITLE
minetest.register_on_placenode not working w/ trees

### ISSUE THREAD

**Issue by mixrat** [minetest/minetest#5807]
**`minetest.register_on_placenode`** doesn't get executed when placing trees (tree stumps). It works with all other default blocks but not with any type of tree.

Example: Run this and place a tree, then place any other node...
```
minetest.register_on_placenode(function(pos, newnode, placer, oldnode, itemstack, pointed_thing)
minetest.chat_send_all("you placed something")
end)
```
I have been able to reproduce the issue on a 0.4.15 server and a 0.4.15-dev local client. 

This issue was also [posted in the Minetest Forum](https://forum.minetest.net/viewtopic.php?f=6&t=17666&p=272055#p272055)

My work on a mod has come to a halt because of this. Please help.

**Comment by Dumbeldor** [comment]
No problems when using client side : 
https://github.com/minetest/minetest/blob/master/clientmods/preview/init.lua#L18

**Comment by mixrat** [comment]
Needs to work without client side

**Comment by paramat** [comment]
Related to #1338 and #656

**Comment by SmallJoker** [comment]
Fixed by #6648
